source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/skin_helpers.php

```
@@ -312,9 +317,10 @@ function caBuildSupportContext(array $template, array $allRepositories) {
 		   when the admin marker file exists. Available for plugins too since
 		   neither side does a source-XML round-trip. */
 		if ($diffFeedReady && is_file(CA_PATHS['caAdmin'])) {
```

PHPStan (2.1.54)

[error] Constant CA_PATHS not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)